assets/components/cards/render/CardFooter.jsx

```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import CardMeta from './CardMeta';
 
-function CardFooter({wordCount, pictureAvailable, source, versioncreated}) {
+function CardFooter({wordCount, pictureAvailable, source, versioncreated, audioAvailable,videoAvailable }) {
```

Comment:
Small thing style thing, maybe a space after the comma like the other parameters!